src/main/java/com/tjtanjin/steve/Main.java

```diff
@@ -0,0 +1,40 @@
+package com.tjtanjin.steve;
+
+import java.io.IOException;
+
+import com.tjtanjin.steve.ui.UiHandler;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Steve using FXML.
+ */
+public class Main extends Application {
+
+    private final Steve steve = new Steve();
```

Comment:
STEVE?

Reply:
Thank you for pointing this out. Fixed it in the latest commit! :)